atlasdb-client/src/main/java/com/palantir/atlasdb/util/AtlasDbMetrics.java

```diff
@@ -51,6 +51,15 @@ private AtlasDbMetrics() {}
 
     public static synchronized void setMetricRegistries(MetricRegistry metricRegistry,
             TaggedMetricRegistry taggedMetricRegistry) {
+        if (!metricRegistry.equals(metrics.get())) {
+            log.warn("The MetricsRegistry was re-set to a different value: the previous registry will be ignored"
+                    + " and the metrics will be registered on the new one.");
```

Comment:
This implies that metrics which have already been registered will be copied over which isn't true.  I'd word this as "... and the metrics may be lost"